src/quo2/components/wallet/progress_bar/style.cljs

```diff
@@ -0,0 +1,32 @@
+(ns quo2.components.wallet.progress-bar.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn- border-and-background-color
+  [customization-color]
+  {:light {:pending   {:border-color     colors/neutral-80-opa-5
+                       :background-color colors/neutral-5}
+           :confirmed {:border-color     colors/neutral-80-opa-5
+                       :background-color colors/success-50}
+           :finalized {:border-color     colors/neutral-80-opa-5
+                       :background-color (colors/custom-color customization-color 50)}
+           :error     {:border-color     colors/neutral-80-opa-5
```

Comment:
one thing, is that the error colors use the semantic coloring.
this could be 
```
:error {:border-color colors/neutral-80-opa-5
:background-color (colors/custom-color :danger 50)
```
<img width="284" alt="Screenshot 2023-08-08 at 07 04 18" src="https://github.com/status-im/status-mobile/assets/22799766/fb8a235b-c9e8-43aa-9824-37a9304a801e">
```

Same for confirmed with "success" color. I believe this would align closer how the design team are using this color set.

Reply:
Also, perhaps we need a better name for the method `colors/custom-color` as it takes colors beyond the "custom" colors however it's a similar mechanism. 🤔

Reply:
> :background-color (colors/custom-color :danger 50)

Updated! Thanks, @J-Son89!

> Also, perhaps we need a better name for the method colors/custom-color as it takes colors beyond the "custom" colors however it's a similar mechanism. 🤔

Yes, the components/screen are actually getting the color from this method. Maybe `colors/get-color` (just like `get`/ `get-in` in Clojure)

Reply:
Yeah it's definitely beyond this pr! :) 
We can figure a good approach and then start to implement it, I don't think it's so high priority but will be nice to adjust at some point